_partials/_install-self-hosted-docker-based.mdx

```diff
@@ -19,15 +18,13 @@
 
     If you are running multiple container instances remember to change the port each Docker instance runs on.
 
-
  1. **Connect to a database on your PostgreSQL instance**
```

Comment:
Aren't these the numbers that need to be adjusted?

Reply:
Gatsby does autonumbering. Best practice is to always use 1. This went a bit strange before because I left 2 newlines between the numbers.